binder/postBuild

```diff
@@ -12,9 +12,12 @@ SCALA_VERSION=2.12.8 ALMOND_VERSION=0.4.0
   sh.almond:scala-kernel_$SCALA_VERSION:$ALMOND_VERSION \
   --sources --default=true \
   -o almond
-./almond --install
+./almond --install \
+  --command "java -XX:MaxRAMPercentage=80.0 -jar almond" \
```

Comment:
Did you test this locally somehow? IIUC, this assumes the `almond` JAR is in the current directory when the kernel is launched, which should not be always the case.

Reply:
Yes, I actually figured it out trying locally. I also tested it on binder in my fork. Just run https://mybinder.org/v2/gh/sbrunk/examples/binder-heapspace?urlpath=lab%2Ftree
and then check the memory in a new notebook:
```scala
Runtime.getRuntime.maxMemory / (1024*1024)
```

It looks like the launcher copies itself to `<kernel_dir>/launcher.jar` when `copyLauncher` is enabled and then sets the kernel spec to that:

https://github.com/almond-sh/almond/blob/61756906116aa81d495872f41d679111ba4e0c02/modules/shared/kernel/src/main/scala/almond/kernel/install/Install.scala#L105-L113

So the launcher only has to be named `almond` during **install** time which should be fine.

Reply:
Once we have support for `$JAVA_OPTS`, we can switch to using that instead.

Reply:
Ok. Let's go ahead then…